docs/resources/vkcs_blockstorage_volume.md

```diff
@@ -18,7 +18,7 @@ resource "vkcs_blockstorage_volume" "bs-volume" {
   name = "bs-volume"
   size = 8
   volume_type = "ceph-hdd"
-  availability_zone = "MS1"
```

Comment:
why? if ms1 zone is insufficient for smth?

Reply:
probably pass zone through env variable?

Reply:
Both are sufficient, I used GZ1 for consistency purposes